plugins/modules/lambda_event.py

```diff
@@ -228,6 +244,37 @@ def get_qualifier(module):
 # ---------------------------------------------------------------------------------------------------
 
 
+def set_default_values(module, source_params):
+    _source_params_cpy = copy.deepcopy(source_params)
+
+    if module.params["event_source"].lower() == "sqs":
+        # Default 10. For standard queues the max is 10,000. For FIFO queues the max is 10.
+        _source_params_cpy.setdefault("batch_size", 10)
+
+        if source_params["source_arn"].endswith(".fifo"):
+            if source_params["batch_size"] > 10:
```

Comment:
I don't think `batch_size` is required, so this should either read from `_source_params_cpy["batch_size"]` which has the default set or it should be `source_params.get("batch_size", 10)` so this doesn't raise an error.

Reply:
I used _source_params_cpy instead.